index.html

```diff
@@ -136,6 +138,23 @@ <h2>Kritika Gupta
               </div>
             </div>
           </div>
+          <div class="card-wrapper">
+              
+            <div class="card">
+              
+              <div class="card-image">
+                <img src="img\AirBrush_20200207233444.jpg" alt="kiki2601">
```

Comment:
Hi @vishaaal, alt text doesn't match with your profile. Please make the required changes.

Reply:
ok sure